src/cascadia/TerminalApp/DefaultProfileUtils.cpp

```diff
@@ -15,19 +15,20 @@ static constexpr std::wstring_view PACKAGED_PROFILE_ICON_EXTENSION{ L".png" };
 // - name: the name of the new profile.
 // Return Value:
 // - A Profile, ready to be filled in
-TerminalApp::Profile CreateDefaultProfile(const std::wstring_view name)
+winrt::TerminalApp::Profile CreateDefaultProfile(const std::wstring_view name)
 {
-    const auto profileGuid{ Microsoft::Console::Utils::CreateV5Uuid(TERMINAL_PROFILE_NAMESPACE_GUID,
+    const winrt::guid profileGuid{ Microsoft::Console::Utils::CreateV5Uuid(TERMINAL_PROFILE_NAMESPACE_GUID,
                                                                     gsl::as_bytes(gsl::make_span(name))) };
-    TerminalApp::Profile newProfile{ profileGuid };
-
-    newProfile.SetName(name);
+    winrt::TerminalApp::Profile newProfile{profileGuid};
+    //auto newProfile = winrt::make<winrt::TerminalApp::Profile>();

```

Comment:
hey look dead code

Reply:
Yeah. I guess GitHub hid my earlier comment about this part. This feels wrong. Like, I tried to use winrt::make here and I get a bunch of errors in base (`'implements_type': is not a member of any direct or indirect base class of 'winrt::TerminalApp::Profile'`).

DefaultProfileUtils.h includes Profile.h too.

If I shouldn't go the `winrt::make` route, I'll remove this dead code and the line under though. I just still don't get why this is happening :/

Reply:
Fixed! Turns out `winrt::make` and `winrt::make_self` both take in the impl type, even though they output different things.